driver/stub/stub_write.c

```diff
@@ -141,7 +141,7 @@ process_standard_request(usbip_stub_dev_t *devstub, unsigned int seqnum, usb_csp
 }
 
 static void
-process_class_request(usbip_stub_dev_t *devstub, usb_cspkt_t *csp, struct usbip_header *hdr)
+process_class_request(usbip_stub_dev_t *devstub, usb_cspkt_t *csp, struct usbip_header *hdr, BOOLEAN vendorreq)
```

Comment:
How about change to process_class_vendor?

Reply:
Do you mean to make a seperate function or to change
`BOOLEAN vendorreq`
to 
`BOOLEAN process_class_vendor`

Thanks,
Richard

Reply:
I mean just changing the function name.

Reply:
Oh ok sure will do!

Reply:
I changed it to process_class_vendor_request so it matches the function called inside this function: submit_class_vendor_req. Is this good for you or do you prefer process_class_vendor?

Reply:
Both would be find :) Never mind. 
By the way, you would better to rebase your work to up-to-date master branch(83339b6). Of course, I can rebase it later if you do not.

Reply:
I rebased up to current master. Sorry about that. I didn't realize I was behind.